src/main/java/org/opentripplanner/routing/algorithm/mapping/GraphPathToItineraryMapper.java

```diff
@@ -357,6 +358,19 @@ private static void setLegPathwayFlag(List<Leg> legs, State[][] legsStates) {
         }
     }
 
+    private static void setRoute(List<Leg> legs, State[][] legsStates) {
```

Comment:
Rename and add javadoc

Reply:
Quick question about the rename comment is there a preferred name you would like for this to make sure I stay within convention?  @t2gran

Reply:
I would expect a `Route` to be a transit Line/Route, this seems to be adding references to the PathwayEdge id. I am not to familiar with what those ids are - so a comment on which information is added to the Leg would be nice. I also  see that this is only done for one leg? Why? It might be ok to use the word `Route`, but I was not sure if this was a copy/paste error or not, since nothing else in this method include the word route.

Reply:
Maybe something like `setLegPathwayId`.